response/response.go

```diff
@@ -220,7 +220,7 @@ func (r Response) SetHeaders(headers map[string][]string) error {
 // Unless manually specified, this method will automatically set the
 // Content-Length header in the produced response for convenience.
 
-func (r Response) Exit(status int, body string, headers map[string][]string) {
+func (r Response) Exit(status int, body []byte, headers map[string][]string) {
```

Comment:
Is this a breaking change? I know it’s expected to be the bytes type, but someone may rely on the string type already.
Should we confirm that which version should this be?

Or can we make it more smarter: accept a interface and transform string to bytes instead internally.

Reply:
Good idea. In this way, we gain the ability to use the new interface but also keep the old code working. But we have some function that is returning `string`, that we want to change to `bytes`. The trick does not work with returning values, and if we only do this to parameters, it will be inconsistent.

We do need to inform users about this breaking change.